diff -u ../libelf-0.8.6/lib/sys_elf.h.in lib/sys_elf.h.in
--- ../libelf-0.8.6/lib/sys_elf.h.in	2005-05-21 08:39:26.000000000 -0700
+++ lib/sys_elf.h.in	2007-01-30 18:16:41.000000000 -0800
@@ -113,6 +113,10 @@
 #   define ELF64_R_INFO(s,t)	(((Elf64_Xword)(s)<<32)+((t)&0xffffffffL))
 #  endif /* ELF64_R_SYM */
 
+#  ifndef EM_MSP430
+#   define EM_MSP430 105
+#  endif
+
 #  if __LIBELF64_LINUX
 typedef __libelf_u64_t	Elf64_Addr;
 typedef __libelf_u16_t	Elf64_Half;
